daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/Converter.scala

```diff
@@ -759,6 +759,7 @@ object Converter {
     def toRight(constructor: String, rank: Int, value: SValue): SValue =
       SVariant(scriptIds.damlScript("UserRight"), Name.assertFromString(constructor), rank, value)
     Right(right match {
+      case UserRight.IdentityProviderAdmin => toRight("IdentityProviderAdmin", 0, SUnit)
```

Comment:
This require changle in the Daml code as well. 
However I am not 100% sure how to do it properly. 
Could you crash here (and below) instead, create an issue (eg. "[Daml-script] should handle IdentityProviderAdmin"), assign @nickchapman-da to it, and linked here with 
```TODO: https://github.com/digital-asset/daml/issues/XXXXX```

Reply:
In `Script.daml` modify `UserRight` as follow:
```
data UserRight
  = ParticipantAdmin
  | CanActAs Party
  | CanReadAs Party
  | IdentityProviderAdmin
  deriving (Show, Eq)
```

Reply:
We need to modify also `toUserRight`.

Reply:
@nickchapman-da do you think we need to version daml script code as we have done for [trigger](https://github.com/digital-asset/daml/pull/15298) ?

Reply:
here we go: https://github.com/digital-asset/daml/issues/15857